tasks/stages/update-demo

```diff
@@ -9,7 +9,7 @@ CLONE_DIR="$WORKDIR/bpmn-io-demo"
 # create work dir
 mkdir -p "$WORKDIR"
 
-git clone --depth=1 "https://$GITHUB_AUTH@github.com/$BPMN_IO_DEMO_ENDPOINT.git" "$CLONE_DIR"
+git clone --depth=1 "https://$GH_AUTH@github.com/$BPMN_IO_DEMO_ENDPOINT.git" "$CLONE_DIR"
```

Comment:
Any reason we rename here?

Reply:
GitHub prevents us from setting env variables which start with `GITHUB_`. Cf. https://docs.github.com/en/actions/reference/environment-variables#naming-conventions-for-environment-variables

Reply:
Thanks. Got it.